register_forte_options.py

```diff
@@ -47,17 +49,21 @@ def register_driver_options(options):
                           'The type of computation')
 
     options.add_int(
-        "CHARGE", 0,
+        "NEL", None,
```

Comment:
Does it mean every time the user should specify these keywords?

Reply:
This is a good question and I should put it into the manual. These quantities are relevant to building the StateInfo objects in forte. Charge, multiplicity, etc.  do not have a default because they cannot have one. However, if you run a job via psi4 the code will by default read charge etc. from the wave function object. If charge and other values are specified then they override the values read from psi4::wavefunction. In FCIDUMP there is enough information to reconstruct a StateInfo object so they are also not needed.

Reply:
Thanks, I see those if statements later in the files.